scripts/check_fips_compatibility.py

```
@@ -378,6 +442,127 @@ def print_issue(issue: FipsIssue, show_hints: bool = False) -> None:
+    all_issues.extend(check_requirements_file(Path("requirements-dev.txt")))
+
+    return all_issues
+
+
+def _print_json_output(all_issues: list[FipsIssue]) -> None:
+    """Print issues in JSON format.
+
+    Args:
+        all_issues: List of all FIPS issues to output.
+    """
+    import json  # noqa: PLC0415
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)